src/lib/containers/Login.tsx

```diff
@@ -8,6 +8,8 @@ import {
 } from '../utils/functions/getEndpoint'
 import { GoogleIcon24 } from '../assets/GoogleIcon24'
 
+export const GOOGLE_AUTH_PROVIDER = 'GOOGLE_OAUTH_2_0'
```

Comment:
If we'll have multiple providers would it make sense to have an enum or a map for provides? 
e..g PROVIDERS = {
GOOGLE: 'GOOGLE_OAUTH_2_0',
....
}

Reply:
Absolutely!  When Login is updated to support more than one, this should definitely be a map.  I'll make the change now to set up the next engineer for the enhancement.